packages/@ama-sdk/core/package.json

```diff
@@ -29,9 +29,9 @@
     "build": "yarn nx build ama-sdk-core",
     "build:cjs": "swc src -d dist/cjs -C module.type=commonjs -q",
     "build:esm2015": "swc src -d dist/esm2015 -C module.type=es6 -q",
-    "build:esm2020": "tsc -b tsconfig.esm2020.json",
+    "build:esm2020": "tsc -b tsconfig.build.json",
```

Comment:
`tsconfig.esm2020.json` still exist. Is is used somewhere else or can we remove it?

Reply:
No, the file does not exist. The reason why we got nothing before is that the script `build:esm2020` was never called. (this build was done by the `compile` configuration which used the nx/js:tsc builder - which added the type: module in package.json btw. So I removed nx builder and align the naming with other builds and call this script defined in package.json)

Reply:
I understand. Thanks!